### PR TITLE
Backport changelog

### PR DESCRIPTION
**Pull request by heartsucker**
## Status

Ready for review

## Description of Changes

Toward #4124

Backport the changelog, bump the version number.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,6 +1,74 @@
 # Changelog
```
Comment (emkll) [inline changelog.md]: nit: there's missing 0.13.0~rc1 at the top of the changelog

**Comment by heartsucker** [comment]
@emkell I squashed that, so we should be good to go now.

**Comment by emkll** [comment]
LGTM, thanks @heartsucker

**Comment by heartsucker** [comment]
I'm consistently getting this failure. Any idea what the root cause is? I don't have access to those machines.

```
    TASK [common : Perform safe upgrade to ensure all the packages are updated.] ***
make: *** [ci-go-xenial] Terminated
Warning: Permanently added '35.230.14.89' (ECDSA) to the list of known hosts.

scp: /home/sdci/securedrop-source/junit/*xml: No such file or directory
Too long with no output (exceeded 10m0s)
```

**Pull request merged**